api/tests/organizational-entities/integration/infrastructure/repositories/organization-for-admin.repository.test.js

```diff
@@ -7,7 +7,7 @@ import { ORGANIZATION_FEATURE } from '../../../../../src/shared/domain/constants
 import { OrganizationInvitation } from '../../../../../src/team/domain/models/OrganizationInvitation.js';
 import { catchErr, databaseBuilder, domainBuilder, expect, knex, sinon } from '../../../../test-helper.js';
 
-describe('Integration | Organizational Entities | Infrastructure | Repository | organization-for-admin', function () {
+describe.only('Integration | Organizational Entities | Infrastructure | Repository | organization-for-admin', function () {
```

Comment:
**todo**: on peut supprimer le `.only`

Reply:
Pardon, j'aurais dû remettre le work in progress 😅